tasks/34_max-triplet-sum/max-triplet-sum.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * maxTripletSum - find maximum triplet sum
+ * in the array without duplicates
+ *
+ * @param {Array<Number>} numbers
+ * @return {Number}
+ */
+export function maxTripletSum(numbers) {
+  return maxNSum(3)(numbers);
+}
+
+/**
+ * maxNSum - find maximum n sum
+ * in the array without duplicates
+ *
+ * @param {Number} n
+ * @return {Function}
+ */
+export function maxNSum(n) {
```

Comment:
This function looks pretty neat! Very nice, just a though you could write this pure "functional" way
```javascript
return number => numbers
     .filter((value, index) => index === numbers.indexOf(value))
     .sort()
     .slice(sortedNumbers.length - n)
     .reduce((acc, val) => acc + val, 0)
```

Reply:
Fixed